server/src/main/java/io/littlehorse/server/LHServerListener.java

```diff
@@ -80,113 +81,8 @@
 import io.littlehorse.common.proto.WaitForCommandResponse;
 import io.littlehorse.common.util.LHProducer;
 import io.littlehorse.common.util.LHUtil;
-import io.littlehorse.sdk.common.proto.ACLAction;
-import io.littlehorse.sdk.common.proto.ACLResource;
-import io.littlehorse.sdk.common.proto.AssignUserTaskRunRequest;
-import io.littlehorse.sdk.common.proto.AwaitWorkflowEventRequest;
-import io.littlehorse.sdk.common.proto.CancelUserTaskRunRequest;
-import io.littlehorse.sdk.common.proto.CompleteUserTaskRunRequest;
-import io.littlehorse.sdk.common.proto.DeleteExternalEventDefRequest;
-import io.littlehorse.sdk.common.proto.DeletePrincipalRequest;
-import io.littlehorse.sdk.common.proto.DeleteScheduledWfRunRequest;
-import io.littlehorse.sdk.common.proto.DeleteTaskDefRequest;
-import io.littlehorse.sdk.common.proto.DeleteUserTaskDefRequest;
-import io.littlehorse.sdk.common.proto.DeleteWfRunRequest;
-import io.littlehorse.sdk.common.proto.DeleteWfSpecRequest;
-import io.littlehorse.sdk.common.proto.ExternalEvent;
-import io.littlehorse.sdk.common.proto.ExternalEventDef;
-import io.littlehorse.sdk.common.proto.ExternalEventDefId;
-import io.littlehorse.sdk.common.proto.ExternalEventDefIdList;
-import io.littlehorse.sdk.common.proto.ExternalEventId;
-import io.littlehorse.sdk.common.proto.ExternalEventIdList;
-import io.littlehorse.sdk.common.proto.ExternalEventList;
-import io.littlehorse.sdk.common.proto.GetLatestUserTaskDefRequest;
-import io.littlehorse.sdk.common.proto.GetLatestWfSpecRequest;
-import io.littlehorse.sdk.common.proto.LHHostInfo;
-import io.littlehorse.sdk.common.proto.ListExternalEventsRequest;
-import io.littlehorse.sdk.common.proto.ListNodeRunsRequest;
-import io.littlehorse.sdk.common.proto.ListTaskMetricsRequest;
-import io.littlehorse.sdk.common.proto.ListTaskMetricsResponse;
-import io.littlehorse.sdk.common.proto.ListTaskRunsRequest;
-import io.littlehorse.sdk.common.proto.ListUserTaskRunRequest;
-import io.littlehorse.sdk.common.proto.ListVariablesRequest;
-import io.littlehorse.sdk.common.proto.ListWfMetricsRequest;
-import io.littlehorse.sdk.common.proto.ListWfMetricsResponse;
+import io.littlehorse.sdk.common.proto.*;
```

Comment:
Eduwer is going to be grumpy about this :joy:

The settings to avoid it on VSCode are:
```
    "java.sources.organizeImports.starThreshold": 400,
    "java.sources.organizeImports.staticStarThreshold": 400
```

Reply:
Ok, I can restore that 😅